fvm/evm/emulator/emulator.go

```diff
@@ -200,64 +200,53 @@ func (proc *procedure) commit() error {
 }
 
 func (proc *procedure) mintTo(
-	address types.Address,
-	amount *big.Int,
+	call *types.DirectCall,
 	txHash gethCommon.Hash,
 ) (*types.Result, error) {
-	addr := address.ToCommon()
-	res := &types.Result{
-		GasConsumed: proc.config.DirectCallBaseGasUsage,
-		TxType:      types.DirectCallTxType,
-		TxHash:      txHash,
-	}
+	bridge := call.From.ToCommon()
 
-	// create account if not exist
-	if !proc.state.Exist(addr) {
-		proc.state.CreateAccount(addr)
+	// create bridge account if not exist
+	if !proc.state.Exist(bridge) {
```

Comment:
just wondering why we don't create these "special" accounts as part of evm setup?

Reply:
hmm, we could, I was more leaning towards lazy creation, but we can revisit this in a future PR for all of the special accounts.